spec/features/staff/users_can_create_a_fund_level_activity_spec.rb

```diff
@@ -73,6 +73,32 @@
       expect(activity.extending_organisation).to eql(user.organisation)
     end
 
+    context "when there is an exsiting activity with a nil identifier" do
+      scenario "successfully create a activity" do
+        visit organisation_path(user.organisation)
+        click_on(I18n.t("page_content.organisation.button.create_fund"))
+
+        visit organisation_path(user.organisation)
+        click_on(I18n.t("page_content.organisation.button.create_fund"))
+
+        fill_in_activity_form(level: "fund")
+
+        expect(page).to have_content I18n.t("form.fund.create.success")
+      end
+    end
+
+    context "when there is an exsiting activity with the same identifier" do
```

Comment:
Is this correct? From the commit message it sounds like you shouldn't be able to create an activity with the same (non-nil) identifier?

Reply:
Also you have a typo in "existing" :)

Reply:
Oh, man!! I had this on my list and forgot! :( look down there, that "A-Unique_Identifier" has an underscore so it's passing!

Thank you for spotting!

Reply:
Phew it wasn't me going mad then 🤣